kpi/models/kpi_threshold_range.py

```diff
@@ -49,40 +49,29 @@ class KPIThresholdRange(models.Model):
     _name = "kpi.threshold.range"
     _description = "KPI Threshold Range"
 
+    VALUE_TYPES = [
```

Comment:
if you want to reuse this data, maybe it is better to add to a function model (@api.model) as a hook in order to reuse it